tox.ini

```diff
@@ -29,7 +29,7 @@ commands =
     black {[testenv:framework]src} --check --diff
     flake8 {[testenv:framework]src}
     mypy {[testenv:framework]src}
-    pytest -c ./pytest-framework.ini -n auto
+    pytest -c ./pytest-framework.ini -n auto -m "not run_in_serial"
```

Comment:
As we briefly discussed yesterday, adding the marker here is not strictly necessary - it's handled by a conftest to avoid  the same problems running tests locally, even if the marker is forgotten. I think we can be explicity it in tox though.
https://github.com/ethereum/execution-spec-tests/blob/0767d80e83c67dc9b4c99289daba771f13917e36/src/pytest_plugins/filler/tests/conftest.py#L10-L15

Note this conftest.py has moved in #792.